src/editor.js

```diff
@@ -760,6 +764,15 @@ export class OdooEditor extends EventTarget {
             this._toRollback === UNBREAKABLE_ROLLBACK_CODE ? false : this._toRollback;
         this._checkStepUnbreakable = false;
     }
+    preventHistoryStep() {
+        this._historyActive = false;
+    }
+    restoreHistoryStep(doStep = false) {
```

Comment:
I do not see the necessity to have a parameter because:
1. the only place it is used in within a command, which always has a step at the end of it's execution
2. the parameter just make an instruction at the end of the method that could be simply after calling the function with that parameter, which would make the code more explicit and with one less indirection.